packages/tools/devtools/devtools-core/src/ContainerDevtools.ts

```diff
@@ -83,6 +87,8 @@ export interface ContainerDevtoolsProps extends HasContainerKey {
 	 * provided when initializing the Devtools.
 	 */
 	dataVisualizers?: Record<string, VisualizeSharedObject>;
+
+	dataEditors?: Record<string, EditSharedObject>;
```

Comment:
Let's remove this for now. We can add editing override support to the API surface later. For now, we'll just use the default editors.

I suspect this change will also make it easier to make more items `@internal`.

Reply:
Don't I need dataEditors there even for the default editors?